apisix/cli/config.lua

```diff
@@ -176,6 +176,8 @@ local _M = {
         ["cas-auth"] = "10m",
         ["ocsp-stapling"] = "10m",
         ["mcp-session"] = "10m",
+        status_report_standalone = "10m",
+        status_report = "10m"
```

Comment:
10m is too large, I only recorded the pid and a bool value, 1m should be enough?

Reply:
done